bucket/jcpicker.json

```diff
@@ -0,0 +1,46 @@
+{
+    "version": "5.6",
+    "description": "Color picker and color editor",
+    "homepage": "https://annystudio.com/software/colorpicker/",
+    "license": {
+        "identifier: "Freeware",
+        "url": "https://annystudio.com/info/#tnc"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://annystudio.com/jcpicker.exe",
+            "hash": "55503ed4934df60fc514453d56cd2495753cc980d6d03cc1ab6b55f07206322e"
+        },
+        "32bit": {
+            "url": "https://annystudio.com/jcpicker_32bit.exe#/jcpicker.exe",
+            "hash": "269b0e90a1fc6968e134b25a6031fc62a4fc734936dafa4265387160ae23851b"
+        }
+    },
+    "pre_install": [
+        "'ini', 'txt' | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\jcpicker.$_\")) { New-Item \"$dir\\jcpicker.$_\" | Out-Null }",
+        "}"
+    ],
+    "bin": "jcpicker.exe",
+    "shortcuts": [
+        [
+            "jcpicker.exe",
+            "Just Color Picker"
+        ]
+    ],
+    "persist": [
+        "jcpicker.ini",
+        "jcpicker.txt"
+    ],
+    "checkver": "Latest version:</b> ([\\d.]+)",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://annystudio.com/jcpicker.exe"
```

Comment:
Any possibility of versioned download URLs?

Reply:
I don't see any

Reply:
They share hashes?

Reply:
No